src/classlibnative/inc/nlsinfo.h

```diff
@@ -84,11 +84,6 @@ class COMNlsInfo {
     static BOOL QCALLTYPE InternalGetUserDefaultUILanguage(QCall::StringHandleOnStack userDefaultUiLanguage);
     static BOOL QCALLTYPE InternalGetSystemDefaultUILanguage(QCall::StringHandleOnStack systemDefaultUiLanguage);
 
-// Added but disabled from desktop in .NET 4.0, stayed disabled in .NET 4.5
```

Comment:
Since this will be a breaking change from 1.0.X and 1.1.x, do we want to enable this under a compat switch?

CC @tarekgh